modules/apps/commerce/headless/headless-commerce/headless-commerce-delivery-catalog-impl/src/main/java/com/liferay/headless/commerce/delivery/catalog/internal/resource/v1_0/ProductResourceImpl.java

```diff
@@ -119,14 +119,15 @@ public Product getChannelProduct(
 				"commerceChannelGroupId", commerceChannel.getGroupId()
 			).build();
 
+		searchContext.setAttributes(attributes);
+
 		long[] commerceAccountGroupIds =
 			_commerceAccountHelper.getCommerceAccountGroupIds(
 				_getAccountId(accountId, commerceChannel));
 
 		searchContext.setAttribute(
```

Comment:
@andrea-ale-sbarra can we just unify and put this to the attribues property? So we avoid to set and than change it?  Just easier to read

Reply:
done